crates/turborepo-lib/src/commands/ls.rs

```diff
@@ -114,6 +148,21 @@ impl<'a> RepositoryDetails<'a> {
         for (package_name, entry) in &self.packages {
             println!("  {} {}", package_name, GREY.apply_to(entry));
         }
+    }
+
+    fn json_print(&self) {
+        println!("{}", serde_json::to_string_pretty(&self).unwrap());
```

Comment:
Do we want to unwrap here?

Reply:
Hmm yea good point, the print doesn't return errors here but I should prob handle this a big better. Let me take another look

Reply:
Good call, done!